Aztec/Classes/NSAttributedString/Conversions/AttributedStringParser.swift

```diff
@@ -508,8 +508,9 @@ private extension AttributedStringParser {
             }
         }
         
-        let lastElement = paragraphNodes.last! // There's no scenario where having zero paragraph nodes would make sense.
```

Comment:
Loved the comment that was there before :).

Reply:
Totally!